website/borrel/admin.py

```diff
@@ -74,10 +148,91 @@ class ReservationItemInline(admin.TabularInline):
     )
 
 
+class BorrelReservationResource(resources.ModelResource):
```

Comment:
maybe `submitted_at`, `created_at` and `updated_at` are also nice columns to have